pkg/ssh/ssh.go

```diff
@@ -112,15 +112,15 @@ func (s *sshTunnel) Open() error {
 
 func (s *sshTunnel) Dial(ctx context.Context, network, address string) (net.Conn, error) {
 	if s.client == nil {
-		return nil, errors.New("tunnel is not opened.")
+		return nil, errors.New("tunnel is not opened")
 	}
 	// This Dial method does not allow to pass a context unfortunately
 	return s.client.Dial(network, address)
 }
 
 func (s *sshTunnel) Close() error {
 	if s.client == nil {
-		return errors.New("Cannot close tunnel. Tunnel was not opened.")
+		return errors.New("cannot close tunnel. tunnel was not opened")
```

Comment:
parity with Dial's err message
```suggestion
		return errors.New("cannot close tunnel: tunnel is not opened")
```